distribution/src/main/resources/VERSIONS.md

```diff
@@ -12,8 +12,8 @@ This file lists the recommended components for this Service Pack. Use it along w
 | Alfresco Search Services | 1.2.0 | 1.1.1 |
 | Alfresco Insight Engine | 1.0.0 |
 | Alfresco Desktop Sync Service | @alfresco.desktop-sync.version@ |
-| Alfresco Transform Service | 0.1.1 EA |
-| Document Transformation Engine | @alfresco.transformation-engine.version@ |
```

Comment:
DTE is a separate product - please revert this line !!

Reply:
I see engine, I update. Reverted 😃